language_generators/scala-defref-extractor/src/test/scala/io/bazeltools/buildgen/scaladefref/CanParseFileTest.scala

```diff
@@ -58,11 +58,7 @@ class CanParseFileTest extends AnyFunSuite {
         Entity.dotted("com"),
         Entity.dotted("com.example"),
         Entity.dotted("com.example.Elephant"),
-        Entity.dotted("com.example.Wolf"),
-        Entity.dotted("com.foo.bar.com"),
-        Entity.dotted("com.foo.bar.com.example"),
-        Entity.dotted("com.foo.bar.com.example.Elephant"),
-        Entity.dotted("com.foo.bar.com.example.Wolf")
+        Entity.dotted("com.example.Wolf")
```

Comment:
This is fun to see that no new tests are necessary, but the change naturally cleans up a bunch of undesirable lines in the existing tests.